src/ConstraintViolationException.php

```diff
@@ -28,24 +28,14 @@ public function isClientSafe(): bool
         return true;
     }
 
-    /**
-     * Returns string describing a category of the error.
-     *
-     * Value "graphql" is reserved for errors produced by query parsing or validation, do not use it.
-     */
-    public function getCategory(): string
```

Comment:
`ClientAware` no longer has this method now, see https://github.com/webonyx/graphql-php/pull/830 for why they removed the category